### PR TITLE
Update README and setup for automation tests

### PR DESCRIPTION
**Pull request by ashuka24**
This commit does the following:
- Update README
- Add --no-psqlrc flag to for easier testing
- Add function that creates the pxf extension when running automation
  testing

Co-authored-by: Ashuka Xue <axue@vmware.com>
Co-authored-by: Bradford Boyle <bradfordb@vmware.com>

### CODE REVIEW

File: automation/tincrepo/main/mpp/lib/PSQL.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def __init__(self, sql_file = None, sql_cmd = None, out_file = None, output_to_f
             if not os.path.exists(sql_file):
                 raise PSQLException('SQL file %s does not exist. ' %sql_file)
 
-            cmd_str = '%s psql %s %s %s %s %s -f %s' \
+            cmd_str = '%s psql %s %s %s %s %s --no-psqlrc -f %s' \
```
Comment (denalex) [inline automation/tincrepo/main/mpp/lib/PSQL.py:57]: why was this needed ?

File: automation/src/main/java/org/greenplum/pxf/automation/components/gpdb/Gpdb.java [hunk 2]
```diff
@@ -71,6 +71,9 @@ public void init() throws Exception {
 
 		connect();
 
+		// Create the extensions if they don't exist
+		createExtension("pxf", true);
```
Comment (denalex) [inline automation/src/main/java/org/greenplum/pxf/automation/components/gpdb/Gpdb.java:75]: Usually we do it once per local setup, but guess this does not hurt. Did we run this via dev pipeline, especially on multinode clusters with upgrade tasks to make sure it doesn't impact the tests ?

**Review by denalex — APPROVED** [review]

File: automation/tincrepo/main/mpp/lib/PSQL.py (hunk 1 above)
Comment (bradfordb-vmware) [reply automation/tincrepo/main/mpp/lib/PSQL.py:57]: If you have any customization in your local psqlrc file (for example always printing timing information) that can cause automation to fail when running locally. The printed timing info shows up in the diff resulting in a tinc failure.

File: automation/src/main/java/org/greenplum/pxf/automation/components/gpdb/Gpdb.java (hunk 2 above)
Comment (bradfordb-vmware) [reply automation/src/main/java/org/greenplum/pxf/automation/components/gpdb/Gpdb.java:75]: I don't know if a dev pipeline has been set but I can make sure one is run before merging.
